### DISCUSSION TITLE
Response Time

### DISCUSSION THREAD

**Discussion by scgrant327** [weather-gov/api#344] [Q&A]
The API has been super slow in responding... most of the time I'm getting:

"An error occurred.
Sorry, the page you are looking for is currently unavailable.
Please try again later.

If you are the system administrator of this resource then you should check the error log for details.

Faithfully yours, nginx."

... for all requests, even the examples on the website.

**Comment by StephenClouse** [comment]
Please see https://weather-gov.github.io/api/reporting-issues

We cannot respond to outages here.
